backend/inspirehep/orcid/tasks.py

```diff
@@ -268,3 +269,13 @@ def _find_user_matching(orcid, email):
     if user_identity:
         return user_identity.user
     return User.query.filter_by(email=email).one_or_none()
+
+
+@shared_task(max_retries=3)
```

Comment:
``max_retries`` alone I think will do nothing you have to ``self.retry`` with ``bind=True`` or you have to specify ``autoretry_for`` and specify the exception (i.e. can be something silly like ``autoretry_for=(Exception,)``) do we really want this?

Reply:
yeah this one is from a copy-paste, tbh I don't think we need retries for this